components/constellation/tracing.rs

```diff
@@ -219,9 +219,9 @@ mod from_script {
                 Self::WebResourceRequested(..) => target_variant!("WebResourceRequested"),
                 Self::AllowUnload(..) => target_variant!("AllowUnload"),
                 Self::Keyboard(..) => target_variant!("Keyboard"),
-                Self::ClearClipboardContents(..) => target_variant!("ClearClipboardContents"),
-                Self::GetClipboardContents(..) => target_variant!("GetClipboardContents"),
-                Self::SetClipboardContents(..) => target_variant!("SetClipboardContents"),
+                Self::ClearClipboard(..) => target_variant!("ClearClipboardContents"),
```

Comment:
```suggestion
                Self::ClearClipboard(..) => target_variant!("ClearClipboard"),
```

same for the two below